chemprop/models/multi.py

```diff
@@ -22,6 +23,7 @@ def __init__(
         init_lr: float = 1e-4,
         max_lr: float = 1e-3,
         final_lr: float = 1e-4,
+        input_scalers: dict[str, StandardScaler] | None = None,
```

Comment:
I think this will be a `dict[str, list[StandardScaler]`? I think we should merge #730 first to fix making the input scalers for multicomponentMPNNs and then check this PR again for correct saving and loading of those scalers.